22.01

HadleyKing opened the pull request.

mr-c commented on content/provenance-domain.md:
```diff
@@ -70,7 +70,7 @@ BCO versioning should adhere to semantic versioning. Given the above conditions
 ### 2.1.3 Review "review"
 
-This is an array to hold reviewer identifiers and a description of the status of an object in the review process. The subtype `reviewer` contains field(s) for name, affiliation, email, and the contribution type of the reviewer. To further record author information, ORCID IDs are included as they allow for the author to curate their information after submission. ORCID identifiers must be valid and ...
+This is an array to hold reviewer identifiers and a description of the status of an object in the review process. The subtype `reviewer` contains field(s) for name, affiliation, email, and the contribution type of the reviewer. To further record author information, ORC IDs are included as they allow for the author to curate their information after submission. ORCID identifiers must be valid and m ...
```
ORCID should always be one word

mr-c reviewed (commented):
Is there a summary of changes since the last version?

mr-c commented on content/top-level.md:
`````diff
@@ -47,10 +47,25 @@ The version of the BCO specification used to define the BCO. It is recomended th
+| ------------ | ---------------------- | ---------------- | -------------- |
+|   https://   | biocomputeobject.org/  |   /BCO_000001    |     /1.5       |
+````
+1) Protocol: [see Scheme in RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986#section-3.1)
`````
`SHOULD` or `MUST` this be HTTPS?

HadleyKing replied:
I left it ambiguous but maybe `RECOMMENDED` HTTPS?

HadleyKing commented:
> Is there a summary of changes since the last version?

## What's Changed
* 1.4.0 by @HadleyKing in https://github.com/biocompute-objects/BCO_Specification/pull/84
* 2.0.0 by @HadleyKing in https://github.com/biocompute-objects/BCO_Specification/pull/92
* Refactoring for Portal linking by @HadleyKing in https://github.com/biocompute-objects/BCO_Specification/pull/93
* update dev by @HadleyKing in https://github.com/biocompute-objects/BCO_Specification/pull/94
* Dev by @HadleyKing in https://github.com/biocompute-objects/BCO_Specification/pull/95


**Full Changelog**: https://github.com/biocompute-objects/BCO_Specification/compare/1.4.2...22.01

HadleyKing commented:
@mr-c Its a lot since we stoped tracking versions once we realized that [IEEE-2791-2020](https://opensource.ieee.org/2791-object/ieee-2791-schema) would be the repo for the standard. This repo has been repurposed to house the documentation, and so this PR will be the first release to be inline with the BCO Portal and DB release.

kee007ney reviewed (approved):
Minor changes made.

I know that this came up before, but why is the external references field only for publicly accessible data, forcing non-public data to find a home somewhere else?

HadleyKing commented:
> Minor changes made.
> 
> I know that this came up before, but why is the external references field only for publicly accessible data, forcing non-public data to find a home somewhere else?

@kee007ney where do you see that? Actually the `external_data_endpoints` should also be used for non public data. I think that was the original intention. So a consumer knows what access privileges they need.

jpat1546 reviewed (approved):
Made minor changes to text.
